Assets/Script/Menu/Navigation/Navigator.cs

```diff
@@ -99,8 +111,34 @@ private void Update()
                 }
             }
 
-            // TODO: Keyboard inputs for menus
-            // UpdateKeyboardInput();
+            // Process keyboard inputs for menus
+            UpdateKeyboardInput();
+        }
+
+        private void UpdateKeyboardInput()
+        {
+            // Get current keyboard
+            Keyboard keyboard = Keyboard.current;
+
+            // If there is no keyboard, bail
+            if (keyboard == null) { return; }
```

Comment:
Code style: don't put single statements on the same line as `if`s (braces optional).
```suggestion
            if (keyboard == null)
            {
                return;
            }
```

Some of us used to do this, but we discourage it now since it mildly obscures the flow of code enough to make reading it more difficult (though I think the braces help with that a little); just a matter of keeping each line to only one action. (I think we still need to update the Rider config to change it to this.)